src/Temporalio/Testing/TestServerOptions.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using System.Collections.Generic;
+
+namespace Temporalio.Testing
+{
+    /// <summary>
+    /// <b>Unstable</b> options for a time-skipping workflow environment.
+    /// </summary>
+    /// <remarks>
+    /// <b>WARNING: This API is subject to change/removal</b>
+    /// </remarks>
+    public class TestServerOptions : ICloneable
+    {
+        /// <summary>
+        /// Gets or sets the existing executable path for the test server.
+        /// </summary>
+        public string? ExistingPath { get; set; }
+
+        /// <summary>
+        /// Gets or sets the version to version of the test server to download.
```

Comment:
Might want to mention which values are valid here (`vN.N.N` and `latest`)

Reply:
And `default`. Was ok not to mention it in Python :-) I will mention it here.

Reply:
Not sure I mentioned it in TS either TBH, only noticed it here.

Reply:
Fixed